### PR TITLE
docs(aio): add mhartington to gde

### PR DESCRIPTION
**Pull request by mhartington**
## PR Type
Adds Mike Hartington to GDE list
cc @StephenFluin 

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[x] Other... Please describe: Adding JSON data for a new GDE to AIO contributors page
```

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -591,5 +591,15 @@
...
+
+  "mhartington" :{
+    "name": "Mike Hartington",
+    "picture": "mhartington.png",
+    "twitter": "mhartington",
+    "website": "https://mhartington.io",
+    "bio": "Mike is a Developer Advocate for the Ionic Framework as well as a GDE in Angular. He spends most of his time making fast PWAs and exploring emerging web standards. When not behind a keyboard, you'll probably find him with a guitar and beer",
+    "group": "GDE"
+
+ }
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: Wrong indentation 😱

File: aio/content/marketing/contributors.json [hunk 2]
```diff
@@ -591,5 +591,15 @@
...
+  },
+
+  "mhartington" :{
+    "name": "Mike Hartington",
+    "picture": "mhartington.png",
+    "twitter": "mhartington",
+    "website": "https://mhartington.io",
+    "bio": "Mike is a Developer Advocate for the Ionic Framework as well as a GDE in Angular. He spends most of his time making fast PWAs and exploring emerging web standards. When not behind a keyboard, you'll probably find him with a guitar and beer",
+    "group": "GDE"
+
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: Unnecessary empty line 😱

File: aio/content/marketing/contributors.json [hunk 3]
```diff
@@ -591,5 +591,15 @@
     "website": "http://ngnir.life/",
     "bio": "Nir is a Principal Frontend Consultant & Head of the Angular department at 500Tech, Google Developer Expert and community leader. He organizes the largest Angular meetup group in Israel (Angular-IL), talks and teaches about front-end technologies around the world. He is also the author of two books about Angular and the founder of the 'Frontend Band'.",
     "group": "GDE"
-  }
+  },
+
+  "mhartington" :{
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: Wrong indentation 😱

File: aio/content/marketing/contributors.json [hunk 4]
```diff
@@ -591,5 +591,15 @@
     "website": "http://ngnir.life/",
     "bio": "Nir is a Principal Frontend Consultant & Head of the Angular department at 500Tech, Google Developer Expert and community leader. He organizes the largest Angular meetup group in Israel (Angular-IL), talks and teaches about front-end technologies around the world. He is also the author of two books about Angular and the founder of the 'Frontend Band'.",
     "group": "GDE"
-  }
+  },
+
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: Unnecessary empty line.

File: aio/content/marketing/contributors.json [hunk 5]
```diff
@@ -592,6 +592,16 @@
     "bio": "Nir is a Principal Frontend Consultant & Head of the Angular department at 500Tech, Google Developer Expert and community leader. He organizes the largest Angular meetup group in Israel (Angular-IL), talks and teaches about front-end technologies around the world. He is also the author of two books about Angular and the founder of the 'Frontend Band'.",
     "group": "GDE"
   },
+
+  "mhartington" :{
```
Comment (alfaproject) [inline aio/content/marketing/contributors.json]: Wrong colon spacing

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
Fixed it up for you :wink:
Marking for merge...

**Comment by matsko** [comment]
@mhartington please rebase again and make sure it's one commit.

File: aio/content/marketing/contributors.json [hunk 6]
```diff
@@ -602,6 +602,15 @@
     "group": "GDE"
   },
 
+  "mhartington": {
+    "name": "Mike Hartington",
+    "picture": "mhartington.png",
+    "twitter": "mhartington",
+    "website": "https://mhartington.io",
+    "bio": "Mike is a Developer Advocate for the Ionic Framework as well as a GDE in Angular. He spends most of his time making fast PWAs and exploring emerging web standards. When not behind a keyboard, you'll probably find him with a guitar and beer",
```
Comment (jenniferfell) [inline aio/content/marketing/contributors.json]: Minor editorial suggestions: 
Mike is a Developer Advocate for the Ionic Framework and a GDE in Angular. He spends most of his time making fast PWAs and exploring emerging web standards. When not behind a keyboard, you'll probably find him with a guitar and beer.

**Comment by mhartington** [comment]
All fixed up @matsko

**Comment by mhartington** [comment]
ping?

**Review by StephenFluin — APPROVED** [review]

**Pull request closed without merging**